docs/source/glossary.rst

```diff
@@ -97,7 +127,8 @@ Glossary
         Examples include: API, Orchestration, Data Processing, etc.
 
     Lite Mode
-        TODO
+        The SNAP instrument uses :term`detectors<Detector>` comprised of many more :term:`pixels<Pixel>` than are actually required to achieve the target d-spacing resolution.
+        In *Lite* mode, all of the event data will be used, but it will be treated as if sourced from an *effective* pixel representing each 8x8 block of *native* pixels. (See :term:`Native Mode`)
```

Comment:
``` rst
In *Lite* mode, all of the event data will be used, but it will be grouped into *effective* pixels, one for each 8x8 block of *native* pixels. (See :term:`Native Mode`)
```
Something along those lines?